products/sle/main.pm

```diff
@@ -469,7 +469,7 @@ sub load_inst_tests() {
             && sle_version_at_least('12-SP2')
             && is_server()
             && (!is_sles4sap() || is_sles4sap_standard())
-            && install_this_version())
+            && (install_this_version() || install_to_other_at_least('12-SP2')))
```

Comment:
what is listed 3 lines above is also "special" and I don't understand why sle-12-sp2 is mentioned there